docs/source/item-validation.rst

```diff
@@ -141,6 +141,17 @@ as a `dict`:
         OtherItem: '/path/to/otheritem_schema.json',
     }
 
+Keys of `dict` can also be strings matching item class names:
+
+.. code-block:: python
+
+    SPIDERMON_VALIDATION_SCHEMAS = {
+        'DummyItem': '/path/to/dummyitem_schema.json',
+        'OtherItem': '/path/to/otheritem_schema.json',
```

Comment:
Hmm… I don’t like that this is done by name instead of export path. Would you be OK with switching approaches? It is how Scrapy settings handle such situations (in fact, Scrapy settings first supported import paths as strings, and later also added support for actual objects).